RemoteActionMapping/CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ target_compile_definitions(${MODULE_NAME} PRIVATE MODULE_NAME=Plugin_${PLUGIN_NA
 
 target_include_directories(${MODULE_NAME} PRIVATE ../helpers)
 
+target_compile_options(${MODULE_NAME} PRIVATE -Wall -Werror -Wno-error=format-truncation)
```

Comment:
please remove flags from the project files

Reply:
Done